bundle/src/main/java/com/adobe/acs/commons/redirects/servlets/RewriteMapServlet.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * ACS AEM Commons
+ *
+ * Copyright (C) 2013 - 2023 Adobe
```

Comment:
@YegorKozlov you might want to update the year range to include 2024 as current year, too ;)